harvester_e2e_tests/integration/test_networks.py

```diff
@@ -0,0 +1,954 @@
+        check_vm_ip_exists(api_client, unique_name, wait_timeout)
+
+        # Get VM interface ipAddresses
+        code, data = api_client.vms.get_status(unique_name)
+        assert 200 == code, (f"Failed to get specific vm content: {code}, {data}")
+
+        interfaces_data = data['status']['interfaces']
+
+        assert 1 == len(interfaces_data), (
+            f"Failed: get more than one interface: {interfaces_data}"
+        )
+
+        mgmt_ip = interfaces_data[0]['ipAddress']
+
+        # Ping management ip address from Harvester node
+        ping_command = "ping -c 50 {0}".format(mgmt_ip)
+
+        _stdout, _stderr = self.ssh_client(
+            client, vip, node_user, node_password, ping_command, wait_timeout)
+
+        stdout = _stdout.read().decode('ascii').strip("\n")
+
+        assert stdout.find(f"64 bytes from {mgmt_ip}") > 0, (
+            f"Failed to ping VM management IP {mgmt_ip} "
+            f"on management interface from Harvester node")
+
+        # SSH to management ip address and execute command from Harvester node
+        _stdout, _stderr = self.ssh_jumpstart(
+            client, mgmt_ip, vip, node_user, node_password,
+            vm_credential["user"], vm_credential["password"], "ls")
+
+        stdout = _stdout.read().decode('ascii').strip("\n")
+
+        assert stdout.find("bin") == 0, (
+            f"Failed to ssh to VM management IP {mgmt_ip} "
+            f"on management interface from Harvester node")
+
+        # Check should not SSH to management ip address from external host
+        command = ['/usr/bin/ssh', '-o', 'ConnectTimeout=5', mgmt_ip]
+        try:
+            result = subprocess.check_output(
+                command, stderr=subprocess.STDOUT, shell=False, encoding="utf-8")
+        except subprocess.CalledProcessError as e:
+            result = e.output
+
+        assert "connect to host {0} port 22: No route to host".format(mgmt_ip) in result, (
+            f"Failed: Should not be able to SSH to VM management IP {mgmt_ip}"
+            f"on management interface from Harvester node")
+
+        # cleanup VM
+        delete_vm(api_client, unique_name, wait_timeout)
+
+        # Revert AllowTcpForwarding for ssh jumpstart
+
+        self.ssh_client(client, vip, node_user, node_password,
+                        restore_tcp, wait_timeout)
+
+        self.ssh_client(client, vip, node_user, node_password,
+                        restart_ssh, wait_timeout)
+
+    @pytest.mark.p0
+    @pytest.mark.dependency(name="vlan_network_connection")
+    def test_vlan_network_connection(self, api_client, request, client, unique_name,
+                                     image_opensuse, vlan_network, wait_timeout):
+        """
+        Manual test plan reference:
+        https://harvester.github.io/tests/manual/network/validate-network-external-vlan/
+
+
+        Steps:
+        1. Create an external VLAN network
+        2. Create a new VM and add the external vlan network
```

Comment:
The test steps is not align to the manual test plan, what the manual test plan asking:
> Make sure that the network is set to the external VLAN with bridge as the type

which means there will have **only 1 NIC**, and it is external VLAN.